apps/frontend/src/App.tsx

```diff
@@ -1,23 +1,11 @@
 import { MainPage } from './pages/MainPage';
-import { Navigate, Route, Routes } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
 
 export const App = () => {
   return (
     <>
       <Routes>
-        <Route path="/" element={<MainPage />}>
-          <Route path="group" element={<MainPage />}>
-            <Route path=":groupId" element={<MainPage />} />
-          </Route>
-          <Route path="workout" element={<MainPage />}>
-            <Route path=":workoutId" element={<MainPage />} />
-          </Route>
-          <Route path="profile" element={<MainPage />} />
-          <Route path="logs" element={<MainPage />} />
-          <Route path="login" element={<MainPage />} />
-          <Route path="auth/*" element={<MainPage />} />
-        </Route>
-        <Route path="*" element={<Navigate to="/" />} />
+        <Route path="*" element={<MainPage />} />
```

Comment:
This did not really give us anything, other than automatic redirect for pages that does not match, but that is done in the crazy `switch` anyways 🤷